### ISSUE TITLE
Tried to release version 0.3.3 of website and failed

### ISSUE THREAD

**Issue by jonfroehlich** [jonfroehlich/makeabilitylabwebsite#819]
Received this email

> Version in "./docker-compose.yml" is unsupported. You might be seeing this error because you're using the wrong Compose file version. Either specify a version of "2" (or "2.0") and place your service definitions under the `services` key, or omit the `version` key and place your service definitions at the root of the file to use version 1.
> For more on the Compose file format versions, see https://docs.docker.com/compose/compose-file/

Also, I can't view the buildlog.txt here: https://makeabilitylab.cs.washington.edu/logs/buildlog.txt
![image](https://user-images.githubusercontent.com/1621749/68517454-80c66380-023c-11ea-9387-a47a18ab59d7.png)

**Comment by mechanicjay** [comment]
The latest upgrade to Centos 7.7 wiped out the newer version of docker-compose needed to read docker-compose.yml file.  I've installed a newer version docker-compose and was able to build the new image and deploy the updated image.  

This turned out to be a complete disaster as this tag pulls in the docker-compose work and the dockerized postgres work that was done over the summer and never deployed to production.

This resulted in there being no content on the site.  So I rolled the site back to 0.3.2 and redeployed so at least there is content there for now. If we're going to deploy, I'll have to prepare a database export and import before the new image is brought up.
